site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -195,7 +221,7 @@ feedback](#share-feedback) if you need this to be fixed.
 
 RPs are expected to support browsers which don't support FedCM. Users should
 be able to use an existing, non-FedCM sign-in process. Learn more about [how
-sign-in works in the FedCM origin trial](/blog/fedcm-origin-trial#sign-into-rp).
+sign-in works in the FedCM](#sign-into-rp).
 
 ### Setting to enable or disable FedCM {: #user-settings}
```

Comment:
It's worth mentioning "exponential cooldown" in this section. In addition, users can manage FedCM on a per-RP basis. e.g. if a user dismissed the UI, an entry would be added to settings UI as displayed in the screenshot below. Users can reenable FedCM on the RP by either going to settings or clicking on the PageInfo UI (lock icon) and reset the permission.